src/scroll-timeline-base.js

```diff
@@ -213,6 +213,7 @@ export class ScrollTimeline {
 
       // View timeline
       subject: null,
+      timeRange: null,
```

Comment:
This is a property of the animation and not the timeline.

Reply:
Removed,